tests/layers/test_smp_profile.py

```diff
@@ -59,48 +61,46 @@ def uploaded_file(self, session, data_dir):
             (MeasurementType, "derived", [True]),
         ]
     )
-    def test_metadata(self, table, attribute, expected_value, uploaded_file):
+    def test_metadata(self, table, attribute, expected_value):
         # need:
         #  * comments = "Filename: filename"
         self._check_metadata(table, attribute, expected_value)
 
+    @pytest.mark.usefixtures("uploaded_file")
     @pytest.mark.parametrize(
         "data_name, attribute_to_check, filter_attribute, "
         "filter_value, expected",
         [
             ('force', 'value', 'depth', -53.17, [0.331]),
-            (
-                'force', 'comments', 'depth', -53.17,
-                ['Filename: S06M0874_2N12_20200131.CSV'],
-            ),
         ]
     )
     def test_value(
         self, data_name, attribute_to_check,
-        filter_attribute, filter_value, expected, uploaded_file
+        filter_attribute, filter_value, expected
     ):
         self.check_value(
             data_name, attribute_to_check,
             filter_attribute, filter_value, expected,
         )
 
+    @pytest.mark.usefixtures("uploaded_file")
```

Comment:
Is there a reason to do this over leaving it in the args? I find this more difficult to navigate in an IDE

Reply:
Ha .. I am actually the other way around and only would like to pass in the arguments that I use. Makes for shorter method signatures